app/model/lib/submission_process.py

```diff
@@ -131,10 +133,6 @@ def validate_data_file(submission_form, data_file=None):
         if 'Compartment' in df:
             uploaded_compartments = set(df['Compartment'])
 
-            if missing_compartments := expected_compartments.difference(set(df['Compartment'])):
-                compartment_description = ', '.join(missing_compartments)
-                errors.append(f"{sheet_name}: Missing compartment(s): {compartment_description}")
```

Comment:
If a compartment is defined at all, at a global level, this will fail regardless if it's actually assigned to an experiment. This validation is kind of pointless anyway, if the user downloaded the template, they'll have all the compartments they need. So let's just ignore it.